### DISCUSSION TITLE
[Support]: I have to restart Frigate to get again video flow

### DISCUSSION THREAD

**Discussion by pilot1981** [blakeblackshear/frigate#10360] [General Support]
### Describe the problem you are having

Hi,
I have frigate with HA.

all works fine but sometime frigate stop to works and I have to restart it to get again video flow

### Version

0.13.2

### Frigate config file

```yaml
#da usare solo per migrare media
#database:
#  path: /config/frigate.db
detectors:
  cpu1:
    type: cpu
    num_threads: 3
  cpu2:
    type: cpu
    num_threads: 3
##    model:
##      path: "/custom_model.tflite"
#da usare quando ci sono più telecamere attive
#birdseye:
#  enabled: True
#    mode: continuous
mqtt:
  # Required: host name
  host: 192.168.6.31
  # Optional: port (default: shown below)
  port: 1883
  # Optional: topic prefix (default: shown below)
  # WARNING: must be unique if you are running multiple instances
  topic_prefix: frigate
  # Optional: client id (default: shown below)
  # WARNING: must be unique if you are running multiple instances
  client_id: frigate
  # Optional: user
  user: xxxxxx
  # Optional: password
  # NOTE: Environment variables that begin with 'FRIGATE_' may be referenced in {}.
  #       eg. password: '{FRIGATE_MQTT_PASSWORD}'
  password: xxxxxx
  # Optional: interval in seconds for publishing stats (default: shown below)
  stats_interval: 60
cameras:
  # Name of your camera
#  balcone:
#    ffmpeg:
#      input_args: preset-http-jpeg-generic
#   input_args: preset-rtsp-udp
#      inputs:
#        - path: rtsp://user:pass@192.168.4.20:6668/blinkhd
#          roles:
#            - detect
  cancello:
    ffmpeg:
      global_args: -hide_banner -loglevel fatal
#da cambiare in base all'hardware che si usa
#      hwaccel_args: h264_v4l2m2m
#      hwaccel_args: preset-vaapi
#      hwaccel_args: preset-intel-qsv-h265
#        - -c:v
#        - preset-vaapi
#        - preset-intel-qsv-h265
#        - h264_v4l2m2m
      inputs:
#        - path: rtsp://192.168.4.11:554/user=admin&password=xxxxxx&channel=1&stream=0.sdp?
      - path: rtsp://192.168.4.11:554/user=admin&password=af788260&channel=1
        roles:
        - detect
        - rtmp
#        - record
#        - clips
#    detect:
#      enabled: true # <---- disable detection until you have a working camera feed
#      width: 2560 # <---- update for your camera's resolution
#      height: 1440 # <---- update for your camera's resolution
#      fps: 5
#    rtmp:
#      enabled: false
#    motion:
#      mask: 0,1440,2560,1440,2560,0,0,0
#      threshold: 30
#      contour_area: 10
#      lightning_threshold: 0.8
    snapshots:
      enabled: true
      retain:
        default: 30
#    record:
#      enabled: True
#      events:
#          pre_capture: 15
#          post_capture: 5
#          retain:
#              default: 10
#              mode: active_objects
    objects:
      track:
      - person
      - car
      - truck
      - bicycle
      - motorcycle
      - dog
      - cat
```

### Relevant log output

```shell
INFO    : No frames received from cancello in 20 seconds. Exiting ffmpeg...
2024-03-10 14:14:40.495094750  [2024-03-10 15:14:40] watchdog.cancello              INFO    : Waiting for ffmpeg to exit gracefully...
2024-03-10 14:14:40.673670298  [2024-03-10 15:14:40] frigate.video                  ERROR   : cancello: Unable to read frames from ffmpeg process.
2024-03-10 14:14:40.675716904  [2024-03-10 15:14:40] frigate.video                  ERROR   : cancello: ffmpeg process is not running. exiting capture thread
```

### FFprobe output from your camera

```shell
[telecamera cancello] Output #0, mp4, to 'pipe:3':
  Metadata:
    title           : RTSP Session
    encoder         : Lavf58.76.100
  Stream #0:0: Video: h264 (Main) (avc1 / 0x31637661), yuvj420p(pc, bt709, progressive), 2560x1440, q=2-31, 20 fps, 20 tbr, 90k tbn, 90k tbc
  Stream #0:1: Audio: aac (LC) (mp4a / 0x6134706D), 32000 Hz, mono, fltp, 64 kb/s
    Metadata:
      encoder         : Lavc58.134.100 aac
frame=    1 fps=0.0 q=-1.0 size=       1kB time=00:00:00.04 bitrate= 207.0kbits/s speed=1.78e+03x    
[aac @ 0x55f533cbb140] Queue input is backward in time
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

USB

### Network connection

Wired

### Camera make and model

KEVIEW KX400FB-P 4MP

**Comment by NickM-27** [comment]
there are no logs included, you should not override the ffmpeg global args so we can see what the issue is

**Comment by pilot1981** [comment]
> there are no logs included, you should not override the ffmpeg global args so we can see what the issue is



> there are no logs included, you should not override the ffmpeg global args so we can see what the issue is

sorry, I don't understand your answer: how I can respect ffmpeg global args?

I have to change log level detail of frigate or I have to launch ffprobe with different args?

**Reply by NickM-27** [reply]
remove global_args from your config and then restart and get new logs after frigate has been running for a couple minutes

**Comment by pilot1981** [comment]
> remove global_args from your config and then restart and get new logs after frigate has been running for a couple minutes



> remove global_args from your config and then restart and get new logs after frigate has been running for a couple minutes

Hi,
I removed global_args, I attached logs...I notice my Frigate setup lost many events...

[ccab4aaf_frigate-fa_2024-03-15T07-38-07.801Z.log](https://github.com/blakeblackshear/frigate/files/14612319/ccab4aaf_frigate-fa_2024-03-15T07-38-07.801Z.log)

**Reply by NickM-27** [reply]
doesn't show any camera issues, just the coral getting stuck a lot which is usually due to using a bad cable (the cable included with the cable is a bad cable) or a hardware / power issue

**Comment by pilot1981** [comment]
> doesn't show any camera issues, just the coral getting stuck a lot which is usually due to using a bad cable (the cable included with the cable is a bad cable) or a hardware / power issue

Hi,
I buyed this coral USB device: https://www.amazon.it/dp/B07R53D12W?psc=1&ref=ppx_yo2ov_dt_b_product_details

You think these cause events lost?

do you suggest to change USB cable USB-A USB C? you have a specific suggestion?

**Reply by NickM-27** [reply]
> You think these cause events lost?

events will be lost when the USB coral gets stuck and can't be used

you should try different cable and see if things improve

**Comment by pilot1981** [comment]
these logs could be related to this issue?

![image](https://github.com/blakeblackshear/frigate/assets/46350465/4fe5d2e0-91b6-4873-89e7-c9c19e47b2c8)

**Comment by pilot1981** [comment]
Hi,
I tried another/better USB cable but I have same result: it go in stuck every 3 minutes.

https://www.amazon.it/dp/B0BW4YCWJ1?psc=1&ref=ppx_yo2ov_dt_b_product_details

I have to try with a powered USB hub?

**Comment by pilot1981** [comment]
Hi,
I tried to use a powered USB hub, but QNAP don't see Coral usb...
